natural_language_processing/text_generation/alpaca/run.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+import torch
+import transformers
+
+from utils.nlp.alpaca_instruct import AlpacaInstruct
+from utils.pytorch import PyTorchRunnerV2, apply_compile_maybe
+from utils.benchmark import run_model
+
+
+def run_pytorch(model_path, num_runs, timeout, dataset_path, disable_jit_freeze=False, **kwargs):
+
+    def run_single_pass(pytorch_runner, dataset):
+        inputs = encode(dataset.get_input_array())
+        outputs = pytorch_runner.run(1, inputs=inputs.input_ids, max_new_tokens=100)
```

Comment:
So, I think number of tokens do differ across runs? I.e. model outputs n <= 100 tokens. 

This creates a need to reimplement our functions again as it is run() call expecting to receive variable length of workload now (in the place where you hard-coded 1 at the moment). With this kind of models we won't know the magnitude of workload before actually calling run() so we need to add a way to pass length of generated output to runner somehow post run.

Reply:
Otherwise we will have inaccurate throughput calculation in place.